packages/datetime/src/timePicker.tsx

```diff
@@ -375,3 +439,8 @@ function handleKeyEvent(e: React.KeyboardEvent<HTMLInputElement>, actions: IKeyE
         }
     }
 }
+
+/** Event handler that exposes the target element's value as a string. */
+function handleStringChange(handler: (value: string) => void) {
```

Comment:
please remove this. it's my favorite util function, too, but it does not belong here. 

🤔 we should move them from docs-theme to core utils, in a future PR. #helpwanted 😄